Calculators/Lattice-Energy-Calculator/style.css

```diff
@@ -0,0 +1,63 @@
+body {
+  font-family: Arial, sans-serif;
+  background-color: #f2f2f2;
+  background-image: url("https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcQGV5d7tcd_q6PKvynba4QFZW-tPjds6YUDzA&s");
```

Comment:
Download and link the image instead of using URL and make the new commit in this same PR, don't close this or create a new one @TBorundia